src/Service/SynchronizationService.php

```diff
@@ -87,6 +87,8 @@ public function handleFullSync(PaginatedApiDataProviderInterface $dataProvider,
 
         $this->eventbusSyncRepository->updateTypeSync($type, $offset, $dateNow, $remainingObjects === 0, $langIso);
 
+        $this->incrementalSyncRepository->removeIncrementaSyncObjectByType($type);
```

Comment:
at each call?
what does "bool $isFull" function argument is for? For the forced fullsync from the orchestrator, something else?